magento/inventory
# MSI-1783: 500 error when change source name from grid

### Fixed Issues (if relevant)
1. [magento-engcom/msi-1783](https://github.com/magento-engcom/msi/issues/1783): 500 error when change source name from grid

### Manual testing scenarios
1. Create source (your source should be created without latitude and longitude)
2. Change source name from source grid

### Contribution checklist
